amd_openvx_extensions/amd_rpp/source/kernel_rpp.cpp

```diff
@@ -1852,22 +1852,28 @@ VX_API_CALL vx_node VX_API_CALL vxExtrppNode_NopbatchPD(vx_graph graph, vx_image
     return node;
 }
 
-VX_API_CALL vx_node VX_API_CALL vxExtrppNode_SequenceRearrange(vx_graph graph, vx_tensor pSrc, vx_tensor pDst, vx_array newOrder, vx_scalar layout) {
-    vx_node node = NULL;
-    vx_context context = vxGetContext((vx_reference)graph);
-    if(vxGetStatus((vx_reference)context) == VX_SUCCESS) {
-        vx_uint32 dev_type = getGraphAffinity(graph);
-        vx_scalar dev_type_scalar = vxCreateScalar(vxGetContext((vx_reference)graph), VX_TYPE_UINT32, &dev_type);
-        vx_reference params[] = {
-            (vx_reference) pSrc,
-            (vx_reference) pDst,
-            (vx_reference) newOrder,
-            (vx_reference) layout,
-            (vx_reference) dev_type_scalar
-        };
-        node = createNode(graph, VX_KERNEL_RPP_SEQUENCEREARRANGE, params, 5);
-    }
-    return node;
+VX_API_CALL vx_node VX_API_CALL vxExtrppNode_SequenceRearrangeImage(vx_graph graph, vx_image pSrc, vx_image pDst, vx_array newOrder, vx_uint32 newSequenceLength, vx_uint32 sequenceLength, vx_uint32 sequenceCount)
+{
+	vx_node node = NULL;
+	vx_context context = vxGetContext((vx_reference)graph);
+	if(vxGetStatus((vx_reference)context) == VX_SUCCESS) {
+		vx_uint32 dev_type = getGraphAffinity(graph);
+		vx_scalar DEV_TYPE = vxCreateScalar(vxGetContext((vx_reference)graph), VX_TYPE_UINT32, &dev_type);
+		vx_scalar NEWSEQUENCELENGTH = vxCreateScalar(vxGetContext((vx_reference)graph), VX_TYPE_UINT32, &newSequenceLength);
+		vx_scalar SEQUENCELENGTH = vxCreateScalar(vxGetContext((vx_reference)graph), VX_TYPE_UINT32, &sequenceLength);
+		vx_scalar SEQUENCECOUNT = vxCreateScalar(vxGetContext((vx_reference)graph), VX_TYPE_UINT32, &sequenceCount);
```

Comment:
Please change all variables to camel case

Reply:
This is part of Image changes so I have retained the same